src/web3/strategy/strategies/CONFIG.ts

```diff
@@ -5,5 +5,6 @@ export default Object.freeze({
     STRATEGY_NONE: 'Standard (1 Vote = 1 Vote)',
     STRATEGY_POOLY: 'POOLY - ERC721 Weighted',
     STRATEGY_DAOPUNKS: 'DAOPUNKS - ERC721 Weighted',
+    STRATEGY_POAP_GNOSIS: 'POAP - Event gated - gnosis',
```

Comment:
Is this gnosis only? POAP goes to mainnet too iirc

Reply:
yes they have mainnet too. WIll add